extensions/desktop/kde-neon/launcher-specific

```diff
@@ -45,7 +45,7 @@ export VLC_PLUGIN_PATH="$SNAP_DESKTOP_RUNTIME/usr/lib/x86_64-linux-gnu/vlc/plugi
 export QTCHOOSER_NO_GLOBAL_DIR=1
 export QT_SELECT=5
 # qtchooser hardcodes reference paths, we'll need to rewrite them properly
-[ -d "$XDG_CONFIG_HOME/qtchooser" ] || mkdir -p "$XDG_CONFIG_HOME/qtchooser"
+[ -d "$XDG_CONFIG_HOME/qtchooser" ] || ensure_dir_exists "$XDG_CONFIG_HOME/qtchooser"
```

Comment:
I understand that this is just a sync, but I do believe this is redundant?
I think `ensure_dir_exists "$XDG_CONFIG_HOME/qtchooser"` should be sufficient?

Thanks for the updates! :smiley:

Reply:
Well spotted! Fixed.